requirements/common.txt

```diff
@@ -26,7 +26,7 @@ itsdangerous==0.24
 webassets==0.10.1
 wsgiref==0.1.2
 requests==2.8.1
-WTForms-Components==0.9.9
-pytimeparse==1.1.5
 redis==2.10.5
 rq==0.5.6
+WTForms-Components==0.9.9
```

Comment:
nit: if you're feelin nitpicky i might checkout this file since this change is unnecessary